src/widget/form/chatform.cpp

```diff
@@ -185,8 +185,12 @@ void ChatForm::onTextEditChanged()
 
 void ChatForm::onAttachClicked()
 {
-    QStringList paths = QFileDialog::getOpenFileNames(this,
-                                                      tr("Send a file"));
+    QStringList paths = QFileDialog::getOpenFileNames ( this,
+                                                      tr("Send a file"),
+                                                      QDir::homePath(),
+                                                      0,
+                                                      0,
+                                                      QFileDialog::DontUseNativeDialog );
```

Comment:
>My experience with getting build tools working on Windows has been less than ideal.

Surprise, surprise… Nevermind me being ultra sarcastic on those topics! I try to keep my mouth shut. :)